programmers/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,12 @@
+function solution(seoul){
+//solution 이란 함수를 선언하였고 seoul란 변수를 함수 안에 넣어주었다.
+    for(let i=0; i<seoul.length; i++){
+//for 문을 사용하여 변수 i가 seoul이란 배열을 탐색할 수 있게 만들어 주었다.
+        if(seoul[i] === "Kim"){
+//Kim이 배열내에 오직 한번만 나타나므로 seoul[i] == Kim 조건을 주어 Kim을 찾았다.
+            return "김서방은 "+i+"에 있다"
+        }
+//if문을 이용하여 Kim이 seoul배열안에 들어있으면 김서방의 위치를 return 하도록 만들어 주었다.
+    }
+}
```

Comment:
```suggestion
function solution(seoul) {
    return `김서방은 ${seoul.indexOf('Kim')}에 있다`
}

```
위에서 사용하신 indexOf 함수를 사용하면 이렇게 좀 더 간단하게 한 줄로 풀 수 있습니다
위에 코드는  ES6에서 도입된 '템플릿 리터럴' 이라는 문법을 같이 사용했는데  
개발할 때도 유용하게 사용되는 문법이니  공부해보시는 것을 추천드려요 
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Template_literals